test/shop_test.rb

```diff
@@ -37,6 +54,9 @@ def test_add_metafield
   end
 
   def test_events
+    fake "shop"
+    @shop = ShopifyAPI::Shop.current
```

Comment:
Thank @kevinhughes27 Sorry I am new to test/unit(or testing without Rspec).

I intended to drop setup since test_current_with_options_should_return_current_shop needs a different setup than the other tests.  Now I see that it would be better to leave them in setup and just override them for the options test.

I've updated the pull request.